### PR TITLE
airmail

### PR DESCRIPTION
**Pull request by iBanks3**
Hopefully I have done this right, please advise. Thanks

### CODE REVIEW

File: data/Airmail.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          "name": "from",
+          "description": "The recipient."
+        },
+        {
+          "name": "cc",
+          "descritption": "The cc recipient."
+        },
+        {
+          "name:  "bcc",
+          "description": "The bcc recipient."
```
Comment (pietropizzi) [inline data/Airmail.json]: It seems that you have an error in the json here.

File: data/app-template.json [hunk 2]
```diff
@@ -1,27 +0,0 @@
-{
```
Comment (pietropizzi) [inline data/app-template.json]: Please don't delete this file.

File: data/Airmail.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (pietropizzi) [inline data/Airmail.json]: This file name should be lowercase so "airmail.json".

Comment (pietropizzi) [reply data/Airmail.json]: Same for the image file. the name should be the same as the identifier, so "airmail.png"

File: data/app-template.json (hunk 2 above)
Comment (iBanks3) [reply data/app-template.json]: Sorry about that, saw that it wasn’t needed in the gmail setup I was following. How can I go about merging this back in? Copy and Paste? If so, at what line should it be pasted in. Truly sorry if I am messing things up, learning how to do this.

File: data/airmail.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
           "description": "The cc recipient."
         },
         {
-          "name:"  "bcc",
+          "name:"  "bcc"
```
[the remark this answers is not in the document]
Comment (iBanks3) [reply data/airmail.json]: Got ya. Now I see.

File: data/airmail.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        {
+          "name": "cc",
+          "description": "The cc recipient."
+        },
+        {
+          "name":  "bcc",
+          "description": "The bcc recipient."
+              
+            }
```
Comment (pietropizzi) [inline data/airmail.json]: There is still something wrong here. Align everything as it should be and then you’ll probably see that you have too many closing brackets.

File: data/airmail.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "actions": [
+    {
+      "name": "compose",
+      "description": "Compose a new mail",
+      "url": "airmail://x-callback-url/send",
+      "parameters": [
+        {
+          "name": "subject",
+          "description": "Subject of email.",
```
Comment (pietropizzi) [inline data/airmail.json]: This trailing comma has to go

File: data/airmail.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        },
+        {
+          "name": "plainBody",
+          "description": "Body text of email.",
```
Comment (pietropizzi) [inline data/airmail.json]: And this one too. JSON is very strict about this. You can run `npm run lintjson` locally to see if all the JSON is valid

File: data/app-template.json (hunk 2 above)
Comment (pietropizzi) [reply data/app-template.json]: Just a file with the same name and the same content.

**Comment by pietropizzi** [comment]
@iBanks3 Are you having trouble finishing this up?

**Comment by pietropizzi** [comment]
@iBanks3 anything I can help with?

**Comment by pietropizzi** [comment]
@iBanks3 I will take this code and add it myself, ok?

**Pull request closed without merging**